utils/src/test/java/com/cloud/utils/ssh/SshHelperTest.java

```diff
@@ -0,0 +1,131 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+package com.cloud.utils.ssh;
+
+import java.io.InputStream;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import com.trilead.ssh2.ChannelCondition;
+import com.trilead.ssh2.Session;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SshHelperTest {
+
+    // throwSshExceptionIfConditionsTimeout tests
+    @Test(expected = SshException.class)
+    public void throwSshExceptionIfConditionsTimeoutTestTimeout() throws SshException {
+        SshHelper.throwSshExceptionIfConditionsTimeout(ChannelCondition.TIMEOUT);
+    }
+
+    @Test
+    public void throwSshExceptionIfConditionsTimeoutTestClosed() throws SshException {

```

Comment:
@rafaelweingartner the tested method ("com.cloud.utils.ssh.SshHelper.throwSshExceptionIfConditionsTimeout(int)") has to throw an exception only if the connection condition is of timeout ("ChannelCondition.TIMEOUT"). Thus, the test method "com.cloud.utils.ssh.SshHelper.throwSshExceptionIfConditionsTimeout(int)" does expect a SshException exception (with `@Test(expected = SshException.class)`); otherwise, an exception is not expected and those tests check that the the method does not throw the SshException.